Modules/_decimal/_decimal.c

```diff
@@ -1519,7 +1519,7 @@ init_extended_context(PyObject *v)
 #ifdef EXTRA_FUNCTIONALITY
 /* Factory function for creating IEEE interchange format contexts */
 static PyObject *
-ieee_context(PyObject *dummy UNUSED, PyObject *v)
+ieee_context(PyObject *self, PyObject *v)
```

Comment:
```suggestion
ieee_context(PyObject *module, PyObject *v)
```

+ change references in the function. `self` is a bit imprecise and could be thought as a "normal" object. I think it's fine to say that `self` here is the module.